example-workflows/README.md

```diff
@@ -0,0 +1,115 @@
+# Cloud Functions - GitHub Actions
+
+An example [workflow](.github/workflows/cloud-functions.yml) that uses [GitHub Actions][actions] to deploy a
+[Hello World Python app](main.py) to [Cloud Funtions][cloud-functions].
+
+This code is intended to be an _example_. You will likely need to change or
+update values to match your setup.
+
+## Workflow description
+
+For pushes to the `main` branch, this workflow will:
+
+1.  Set up job
+    - Current runner version: '2.275.1'
+    - Operating System (Ubuntu 18.04.5)
+    - Virtual Environment
+    - Prepare workflow directory
+    - Prepare all required actions
+    - Getting action download info
+    - Download action repository 'actions/checkout@v2'
+    - Download action repository 'google-github-actions/setup-gcloud@master'
+    - Download action repository 'google-github-actions/deploy-cloud-functions@main'
+
+2.  Checkout Github Repo
+    - Run actions/checkout@v2
+    - Syncing repository: google-github-actions/setup-gcloud/example-workflows/cloud-functions
+    - Getting Git version info
+    - Setting up auth
+    - Fetching the repository
+    - Checking out the ref
+
+3.  Run google-github-actions/setup-gcloud
+    - Activated service account credentials 
+    - Exported Default Application Credentials
+
+4.  Run google-github-actions/deploy-cloud-functions
+    - Setting project Id from $GCLOUD_PROJECT
+    - Creating a function revision
+    - Creating or Updating function deployment
+
+
+## Setup
+
+1.  Create a new Google Cloud Project (or select an existing project) and
+    [enable the Cloud Functions and Cloud Build APIs](https://console.cloud.google.com/flows/enableapi?apiid=cloudbuild.googleapis.com,run.googleapis.com).
+
+1.  Create or reuse a GitHub repository for the example workflow:
```

Comment:
I'm not a fan of the create a repo and copy format. That doesn't make this a quick example. In the other repos the [example workflows](https://github.com/google-github-actions/deploy-cloudrun#example-workflows) still live in the `.github/workflows` directory with a specific branch push to trigger them.

Reply:
Agreed, and removed this step from Setup section.

Reply:
With this change, for the push to main to kick off the workflow the workflow files needs to be under `.github/workflows/cloud-functions.yml` not `example-workflows/.github/workflows/cloud-functions.yml` and the action needs to be able to find the function in `example-workflows/` using the input `source_dir`.

Reply:
@averi, if I understand your comments correctly, here is what you are suggesting:  
1. Move example-workflows/.github/workflows/cloud-functions.yml to .github/workflows/deploy-cf-example.yml
2. Change the folder "example-workflows" to "example-app"
3. Use source-dir in deploy-cf-example.yml to find the example app in example-app

Thanks!

Reply:
Yes, that would be great!